core/src/test/java/io/github/belgif/rest/guide/validator/core/model/MediaTypeTest.java

```diff
@@ -0,0 +1,64 @@
+package io.github.belgif.rest.guide.validator.core.model;
+
+import io.github.belgif.rest.guide.validator.core.model.helper.MediaType;
+import io.github.belgif.rest.guide.validator.core.model.helper.MediaTypeException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MediaTypeTest {
+
+    @Test
+    void testIncludes() {
+        var mediaType = new MediaType("application/json");
+        assertTrue(mediaType.includes(new MediaType("application/problem+json")));
+        assertTrue(mediaType.includes(new MediaType("application/json")));
+        assertFalse(mediaType.includes(new MediaType("application/problem+xml")));
+        assertFalse(mediaType.includes(new MediaType("*/*")));
+
+        mediaType = new MediaType("multipart/*");
+        assertTrue(mediaType.includes(new MediaType("multipart/*")));
+        assertTrue(mediaType.includes(new MediaType("multipart/form-data")));
+        assertTrue(mediaType.includes(new MediaType("multipart/chunked")));
```

Comment:
In this test and below "*/*" one: remove tests that validate same logic:
* e.g. if form-data and chuncked tests don't trigger different includes logic, it suffices to only test one of them 
* e.g. for "*/*" test: "application/*" ~ "multipart/*", "application/problem+json" ~ "application/json"

Reply:
Not completely sure what you mean by "/" test: But I think the tests are now modified to only test each logic once.